### PR TITLE
remove swagger

### PR DESCRIPTION
**Pull request by nicolasburtey**

### CODE REVIEW

**Review by krtk6160 — CHANGES_REQUESTED** [review]
`test/integration/entrypoint/graphql.spec.ts#L35` has leftover swagger code, but otherwise LGTM

**Comment by bodymindarts** [comment]
Check code still failing @nicolasburtey https://github.com/GaloyMoney/galoy/pull/352/checks?check_run_id=3117887388

**Comment by nicolasburtey** [comment]
yeah I need to add a pre-hook git to run prettier automatically

**Comment by bodymindarts** [comment]
> yeah I need to add a pre-hook git to run prettier automatically

I'd add it on-save to your editor

File: src/mongodb.ts [hunk 1]
```diff
@@ -23,7 +23,7 @@ const path = `mongodb://${user}:${password}@${address}/${db}`
 
 export const setupMongoConnection = async () => {
   // FIXME we may not want to keep this in the logs
-  baseLogger.info({path}, "db connection path")
+  baseLogger.info({ path }, "db connection path")
```
Comment (krtk6160) [inline src/mongodb.ts:26]: I think we should remove this now. It outputs the password in the logs.

**Review by krtk6160 — CHANGES_REQUESTED** [review]

File: src/mongodb.ts (hunk 1 above)
Comment (nicolasburtey) [reply src/mongodb.ts]: unrelated to this PR but I've push the fix here.

Comment (krtk6160) [reply src/mongodb.ts]: Yep, I just suggested because this PR and the requested change were pretty small

**Review by krtk6160 — APPROVED** [review]
LGTM (except for the failing tests)

**Comment by nicolasburtey** [comment]
failed tests is unrelated unfortuantely

**Pull request merged**